packages/node_modules/@ciscospark/plugin-phone/src/call.js

```diff
@@ -1513,6 +1514,25 @@ const Call = SparkPlugin.extend({
       this.on('change:locus', listener);
       listener();
     });
+  },
+
+  /**
+   * Returns a {@link Readable} that emits {@link Call#media.pc}'s
+   * {@link RTCStatsReport} every second.
+   * @returns {StatsStream}
+   */
+  getRawStatsStream() {
+    // TODO stop polling after hangup
```

Comment:
This TODO will be in another PR?

Reply:
actually, it's already done, i just need to remove the TODO